test/Microsoft.AspNet.Mvc.IntegrationTests/FormFileModelBindingIntegrationTest.cs

```diff
@@ -154,9 +154,8 @@ private class Address
 
             // Assert
 
-            // ModelBindingResult
-            Assert.NotNull(modelBindingResult); // Fails due to bug #2456
-            Assert.Null(modelBindingResult.Model);
+            // ModelBindingResult; expect null because binding failed.
+            Assert.Null(modelBindingResult);
```

Comment:
This is wrong, the FormFileModelBinder should return non null MBR, because the type is IFormFile. That is what bug #2456 was about. Because of the quirkiness of the CompositeModelBinder, the result is null. Lets discuss this.